cdap-watchdog/src/test/java/co/cask/cdap/metrics/process/MetricsProcessorServiceTest.java

```diff
@@ -132,15 +129,15 @@ public void testMetricsProcessor() throws Exception {
                                            partitions, messagingService, injector.getInstance(SchemaGenerator.class),
                                            injector.getInstance(DatumReaderFactory.class),
                                            metricStore,
-                                           1);
+                                           1, 50);
     messagingMetricsProcessorService.startAndWait();
 
     // Publish metrics with messaging service and record expected metrics
     for (int i = 10; i < 20; i++) {
       publishMessagingMetrics(i, METRICS_CONTEXT, expected, SYSTEM_METRIC_PREFIX, MetricType.COUNTER);
```

Comment:
This part is publishing metrics to TMS, while the previous part is publishing metrics to Kafka.